css/style.css

```diff
@@ -0,0 +1,177 @@
+@import url('https://fonts.googleapis.com/css?family=Lato&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: 'Lato', sans-serif;
+}
+
+body {
+  position: relative;
+}
+
+section {
+  width: 100vw;
+  margin: 0;
+}
+
+#navbar {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 80px;
+  width: 100vw;
+  background-color: #be3144;
+  display: flex;
+  z-index: 10;
+}
+
+.nav-list {
+  margin-left: auto;
+  display: flex;
+  align-items: center;
+}
+
+.nav-list > li {
+  display: inline-block;
+  margin-right: 25px;
+}
+
+.nav-list a {
+  color: #fff;
+  text-decoration: none;
+  font-size: 1rem;
+}
+
+#welcome-section {
+  top: 80px;
+  height: 100vh;
+  background-color: #2f3034;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+#welcome-section > h1 {
+  color: #fff;
+  font-size: 2rem;
+  margin-bottom: 5px;
+}
+
+footer > p {
+  color: #fff;
```

Comment:
- [ ] [REQUIRED]  It will be nice if you don't repeat the same CSS rules. DRY and Immutable code is your endpoint in developing your Infrastructure-as-Code deployments. Working towards it will improve your environment, making your code more readable and manageable.